test/components/opt-in-flyout/utilities.js

```diff
@@ -0,0 +1,21 @@
+const toArray = function(collection) {
+	if (Array.from) {
+		return Array.from(collection);
+	}
+
+	const array = [];
+	for (let i = 0; i < collection.length; i++) {
+		array.push(collection[i]);
+	}
+	return array;
+};
+
+export default {
```

Comment:
I'm going to revisit this when I add the vdiff tests. At the very least `toArray` can be replaced by the spread operator.